dissect/target/plugins/os/unix/log/auth.py

```diff
@@ -0,0 +1,69 @@
+from dissect.target.helpers.fsutil import TargetPath, decompress_and_readlines
+from dissect.target.helpers.record import TargetRecordDescriptor
+from dissect.target.plugin import Plugin, export
+
+from datetime import datetime
+import re
+
+AuthLogRecord = TargetRecordDescriptor(
+    "linux/log/auth",
+    [
+        ("datetime", "ts"),
+        ("string", "msg"),
+    ],
+)
+
+
+class AuthPlugin(Plugin):
+    def check_compatible(self):
+        return self.target.fs.path("/var/log/auth.log").exists() or self.target.fs.path("/var/log/secure").exists()
+
+    @export(record=[AuthLogRecord])
+    def securelog(self):
+        return self.authlog()
+
+    @export(record=[AuthLogRecord])
+    def authlog(self):
+        """
+        Yields AuthLogRecords from /var/log/auth.log* and /var/log/secure*
+        """
```

Comment:
```suggestion
        """Return contents of /var/log/auth.log* and /var/log/secure*."""
```
Just for consistency sake with the `messages` plugin.